README.md

```diff
@@ -24,6 +24,61 @@ cd $(The repo you just cloned)
 
 ## 2. Install all gems
```

Comment:
I know it's not really related to this PR, but looking at this readme in general, we kind of just launch into instructions without clarifying that the instructions are to build the website locally and view it. Should we mention that before `1. Clone the repo` or is it not important? Just a thought I had!

Reply:
this is a great point @nkorinek and I was confused by that too when I created it locally. just adding a couple sentence blurb here about the purpose would be helpful.

Reply:
Thank you both for this feedback!! i really appreciate it!!